Automattic/jetpack
# Add tooling for building the jetpack forms dashboard

## Proposed changes:

This patch adds the necessary tooling and setup to continue work on the jetpack forms dashboard refresh, originally started in #27071.

It's currently set up to have a separate submenu entry under 'Jetpack' which will make it easy to work on it and chances are that's where it will eventually end up anyway.  
That said, we'll likely initially allow users to access the old dashboard anyway until we've reached feature parity but setting that up goes beyond the scope of this PR.

### Other information:


## Jetpack product discussion

pcsBup-Fk-p2

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

You will need to enable the `automattic/jetpack-forms` package as well as the new dashboard before testing, by making sure these two filters are added before the contact form module is loaded:

```php
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
add_filter( 'jetpack_forms_dashboard_enable', '__return_true' );
```

Make sure the package is installed with `jetpack install packages/forms && jetpack install plugins/jetpack`, and built the JS bundle with `jetpack build packages/forms`. You can also test the `watch` functionality with `jetpack watch packages/forms`.

- Go to WP Admin.
- There should be a new 'Jetpack Forms' submenu item inside 'Jetpack'.
- When you proceed clicking on it, you should see the new dashboard. Not it's not much for now ;)
